libs/core/kiln_ai/datamodel/extraction.py

```
@@ -1,11 +1,16 @@
+import logging
+import mimetypes
```

Ruff (0.11.9)

`mimetypes` imported but unused

Remove unused import: `mimetypes`

(F401)